.codeclimate.yml

```diff
@@ -1,15 +1,167 @@
-engines:
-  golint:
+# This is our default .CodeClimate.yml, broken out by language. Uncomment the
+# sections at the bottom that apply to your project. ACTION comments indicate
+# places where config might need to be tweaked.
+
+version: "2"
+
+plugins:
+
+# ---------------
+# Cross-language plugins. Should always be on.
+
+  duplication: # Looks for similar and identical code blocks
     enabled: true
-    exclude_patterns:
-    - "**/"                # exclude all
-    - "!./cmd/"            # unexclude just the ones we want to lint
-    - "!./pkg/"
-    - "!./templates/"
+    config:
+      languages:
+        go:
+        java:
+        javascript:
+        php:
+        python:
+          python_version: 3  # ACTION Comment this out if using Python 2
+        ruby:
+        swift:
+        typescript:
 
+  fixme: # Flags any FIXME, TODO, BUG, XXX, HACK comments so they can be fixed
+    enabled: true
+    config:
+      strings:
+      - FIXME
+      - TODO
+      - HACK
+      - XXX
+      - BUG
+
+# ---------------
+# Commonly-used languages - run time is minimal and all of these will work
+# whether files of that language are found or not. In general, leave uncommented
+
+  # Markdown
+  markdownlint:
+    enabled: true
+
+  # Go
   gofmt:
     enabled: true
+    exclude_patterns:
+      - "**/"         # exclude all
+      - "!./cmd/"     # ynexclude just the ones we want to lint
+      - "!./pkg/"
+      - "!./templates"
+  golint:
+    enabled: true
+  govet:
+    enabled: true
 
+  # Ruby
```

Comment:
All of the ruby checkers should get removed

Reply:
See above